public/src/components/drawer.tsx

```diff
@@ -0,0 +1,186 @@
+import React from 'react';
+import clsx from 'clsx';
+import Drawer from '@material-ui/core/Drawer';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemText from '@material-ui/core/ListItemText';
+import {Link} from "react-router-dom";
+import IconButton from "@material-ui/core/IconButton";
+import MenuIcon from "@material-ui/icons/Menu";
+import makeStyles from "@material-ui/core/styles/makeStyles";
+import RRControlPanelLogo from "./rrControlPanelLogo";
+
+const useStyles = makeStyles({
+  list: {
+    width: '20vw',
+    minWidth: '260px'
+  },
+
+  drawerHeading: {
+    top: '0',
+    left: '0',
+    width: '100%',
+    minHeight: '64px',
+    paddingTop: '10px',
+    paddingLeft: '10px',
+    marginBottom: '12%',
+    boxSizing: 'border-box',
+    borderBottom: '1px solid #616161'
+  },
+
+  link: {
+    textDecoration: 'none',
+    color: '#616161'
+  },
+
+  disabledListItem: {
+    borderTop: '1px solid #E0E0E0',
+    marginLeft: '15%',
+    width: '85%',
+    '&:after': {
+      content: ' (disabled)'
+    }
+  },
+
+  listItem: {
+    borderTop: '1px solid #E0E0E0',
+    marginLeft: '15%',
+    width: '85%',
+    '&:hover': {
+      backgroundColor: '#ededed'
+    },
+    '&:focus': {
+      backgroundColor: '#dcdcdc'
+    },
+  },
+
+  menuButton: {
+    marginRight: 20,
+  },
+
+  fullWidthRule: {
+    width: "100%",
+    borderTop: "1px solid #616161"
+  },
+
+  sectionHeader: {
+    borderTop: '1px solid #616161',
+    width: '100%',
+    paddingLeft: '5%',
+    paddingTop: '7%',
+    marginTop: '13%',
+    fontFamily: 'Roboto',
+    fontStyle: 'normal',
+    fontWeight: 'normal',
+    fontSize: '18px',
+    lineHeight: '20px'
+  },
+
+  topSectionHeader: {
+    width: '100%',
+    paddingLeft: '5%',
+    paddingTop: '7%',
+    fontFamily: 'Roboto',
+    fontStyle: 'normal',
+    fontWeight: 'normal',
+    fontSize: '18px',
+    lineHeight: '20px'
+  }
+
+});
+
+const anchor = 'left';
+
+export default function NavDrawer() {
+  const classes = useStyles();
+  const [state, setState] = React.useState({
+    left: false
+  });
+
+  const toggleDrawer = (anchor: string, open: boolean) => (
+    event: React.KeyboardEvent | React.MouseEvent,
+  ) => {
+    if (
+      event.type === 'keydown' &&
+      ((event as React.KeyboardEvent).key === 'Tab' ||
+        (event as React.KeyboardEvent).key === 'Shift')
+    ) {
+      return;
+    }
+
+    setState({ ...state, [anchor]: open });
+  };
+
+  const list = (anchor: string) => (
+    <div
+      className={clsx(classes.list)}
```

Comment:
what is this doing?

Reply:
@JoemG any idea re. the above?

Reply:
https://stackoverflow.com/questions/57557271/understanding-usage-of-clsx-in-react

Looks like it might not be doing anything in this instance as there is no conditional. I will see if it works without it and if so take it out.